editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -17,6 +18,19 @@ function MapAndLabelComponent(props: Props) {
         policyRef={props.policyRef}
         howMeasured={props.howMeasured}
       />
+      <MapContainer environment="standalone">
+        <my-map
+          drawMode
+          drawPointer="crosshair"
+          zoom={20}
+          // adding 1A to the hexcode string adds 10% opacity to the colour
+          drawFillColor={props.drawColour + "1A"}
```

Comment:
Here, I tried to use the ``lighten`` method from material UI Core but it kept returning an rgb string, ``rbg(10, 10, 10)`` which kept throwing an error. I could then pass it through a rgbToHex fn but it seemed more complicated, and maybe the note creates the clarity on what's happening?

Reply:
Following this, I done more research and found ``alpha()`` which returned an `rgba(10, 10, 10) `string which is being accepted by the `<my-map>` component, I'll move forward with this approach